src/testResult.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  RawResult,
+  RawNodeResult,
+  TestResult,
+  Selector
+} from './types'
+
+export default function testResult(
+  { description }: RawResult,
+  outcomes: RawNodeResult[] = []
+): TestResult[] {
+  return outcomes.map(
+    ({ node, result }): TestResult => {
+      return {
+        '@type': 'TestResult',
+        info: description,
+        outcome: 'earl:' + result,
+        pointer: cssToPointer(node.selector)
+      }
+    }
+  )
+}
+
+export function cssToPointer(selector: Selector): string {
+  if (Array.isArray(selector[0])) {
+    return selector[0][0]
+  }
+  // @ts-ignore:
```

Comment:
Why are you doing this?

Reply:
If there are no pre-existing types, should write custom types, instead of ignoring.

Reply:
Modules come into play, when typing a 3rd party library vs our own source code. It is best practise to nest interfaces into relevant modules, to avoid namespace confusion.

https://www.typescriptlang.org/docs/handbook/modules.html